stapi_fastapi/__dev__.py

```diff
@@ -34,8 +34,19 @@ def validate_backend(cls, value: str):
 
 settings = DevSettings()
 backend = settings.backend_name()
+
+# Compose the router
+main_router = MainRouter()
+main_router.include_router(umbra_spotlight_router, prefix="/umbra_spotlight")
+
 app = FastAPI(debug=True)
-app.include_router(StapiRouter(backend).router)
+app.include_router(main_router)
+
+# Define a root endpoint
+# TODO: do I need this?
```

Comment:
This should an endpoint on the `MainRouter` and should not be redeclared here. It should follow [the specification for the Landing Page](https://github.com/stapi-spec/stapi-spec?tab=readme-ov-file#landing-page). That said, we don't need to tackle the landing page as part of this PR; you could remove this endpoint and we could simply make an issue to add landing page support to the `MainRouter`.

Reply:
I see that the `MainRouter` does have a root route, so I think you can just remove this and all will be well.